src/components/LibrariesListItem.tsx

```diff
@@ -9,6 +9,7 @@ import { GenericWedgeIcon } from "@nypl/dgx-svg-icons";
 export interface LibrariesListItemProps {
   library: LibraryData;
   store: Store<State>;
+  current?: boolean;
```

Comment:
This prop is not being set for `LibrariesListItem` so it doesn't look like this PR is needed https://github.com/NYPL-Simplified/reusable-components/pull/12/files

Reply:
It is not needed.  Will update description.

Reply:
This is still here as well as the `openByDefault` prop in `Panel` below.